src/v/redpanda/admin/server.cc

```diff
@@ -2740,17 +2740,20 @@ admin_server::self_test_start_handler(std::unique_ptr<ss::http::request> req) {
                     r.dtos.push_back(cluster::diskcheck_opts::from_json(obj));
                 } else if (test_type == "network") {
                     r.ntos.push_back(cluster::netcheck_opts::from_json(obj));
+                } else if (test_type == "cloud") {
```

Comment:
i'd recommend splitting out the changes to the admin server into a commit the comes right before the rpk changes that use those new http interfaces. makes it much easier to review.